api/src/main/java/org/apache/iceberg/types/Types.java

```diff
@@ -543,6 +565,134 @@ public int hashCode() {
     }
   }
 
+  public static class GeometryType extends PrimitiveType {
+
+    private final String crs;
+
+    private GeometryType(String crs) {
+      if (crs != null) {
+        Preconditions.checkArgument(!crs.isEmpty(), "Invalid CRS: (empty string)");
```

Comment:
why the extra parens around 'empty string'?

Reply:
This is following the existing error message style: https://github.com/apache/iceberg/blob/apache-iceberg-1.8.1/core/src/main/java/org/apache/iceberg/view/BaseView.java#L116, I thought that it is a convention to add parenthesis around empty string.